certification/internal/rpm/rpm.go

```diff
@@ -10,6 +10,9 @@ import (
 	preflightErrors "github.com/redhat-openshift-ecosystem/openshift-preflight/certification/errors"
 )
 
+// GetPackageList returns the list of packages in the rpm database from /var/lib/rpm,
```

Comment:
This is sort of correct... It first looks for rpmdb.sqlite at /var/lib/rpm/rpmdb.sqlite (RHEL 9). If not found, it checks for /var/lib/rpm/Packages (RHEL 8 and prior).

Reply:
Fixed